docs/documentation/concepts/units-of-measurement.md

```diff
@@ -71,3 +71,116 @@ When creating QuantityType states the framework offers some useful packages and
 The `org.eclipse.smarthome.core.library.unit` package contains the classes `SIUnits`, `ImperialUnits` and `SmartHomeUnits` which provide units unique to either of the measurement systems and common units used in both systems.
 The `MetricPrefix` class provides prefixes like MILLI, CENTI, HECTO, etc. which are wrappers to create derived units.
 The `org.eclipse.smarthome.core.library.dimension` and `javax.measure.quantity` packages provide interfaces which are used to type the generic QuantityType and units. 
+
+List of Units
```

Comment:
In one of the first commits this was a headline with `## List...`. Please turn it into a headline again.

Reply:
But @sjka told me to remove that. Ok I do it now

Reply:
Done

Reply:
His comment was about the `Imperial:` headline, see https://github.com/eclipse/smarthome/pull/5840#discussion_r199483562.